flink-python/src/main/java/org/apache/flink/api/common/python/PythonBridgeUtils.java

```diff
@@ -36,6 +38,8 @@
  */
 public final class PythonBridgeUtils {
 
+	private static Pickler pickler = new Pickler();
```

Comment:
Make it an local variable of rexLiteralToPythonObject as I think there is no need to reuse it.